### DISCUSSION TITLE
Disable extension in jupyter notebooks

### DISCUSSION THREAD

**Discussion by jborbik** [vscode-neovim/vscode-neovim#1928] [Q&A]
Is there a way to disable the extension for jupyter notebooks? A quick google search didn't return anything.

**Comment by DJohnston-Cohesity** [comment]
I'm looking for the same thing.  It looks like vscode-neovim allows exclusions only by editorLangId, but that attribute is set based on the cell interpretter and not the notebook file type.  VSCode allows a workspace level enable/disable feature, but it's not controllable in a config file and thus cannot be mapped to a file extension.

I switched to this extension hoping for a less-broken experience when using Notebooks.  It's less broken than 'vscodevim' but I would still prefer to disable the extension.

**Reply by xiyaowong** [reply]
You can bind shortcuts to execute `vscode-neovim.stop` and `vscode-neovim.restart`, both are lightweight commands. You can also write an extension to automatically execute these commands. 

Additionally, vscode-neovim works very well in notebooks, if there are any issues, feel free to raise an issue.
